packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -6,16 +6,20 @@ import Loader from 'react-loader-spinner'
 import { twMerge } from 'tailwind-merge'
 import { BigNumber, utils } from 'ethers'
 import { L1Network, L2Network } from '@arbitrum/sdk'
-import { l2Networks } from '@arbitrum/sdk/dist/lib/dataEntities/networks'
+import {
+  isL1Network,
```

Comment:
We're moving away from using the network objects directly in the UI, so let's expand `isNetwork` in `networks.ts` to have `isEthereum`, which should be the opposite of `isArbitrum`.

Reply:
done